docs/benchmarks.md

```diff
@@ -4,53 +4,51 @@
 
 ```
 BASIC benchmark averages
-Bunyan average: 549.042ms
-Winston average: 467.873ms
-Bole average: 201.529ms
-Debug average: 253.724ms
-LogLevel average: 282.653ms
-Pino average: 188.956ms
-PinoExtreme average: 108.809ms
+Bunyan average: 775.758ms
+Winston average: 673.231ms
+Bole average: 466.211ms
+Debug average: 469.900ms
+LogLevel average: 464.293ms
+Pino average: 383.885ms
+PinoAsync average: 180.870ms
+PinoNodeStream average: 385.438ms
```

Comment:
All of these number hav significantly increased. Did they come from your machine? The GH Action has different output:

https://github.com/pinojs/pino/pull/828/checks?check_run_id=611712858#step:5:172
```
BASIC benchmark averages
Bunyan average: 662.904ms
Winston average: 564.752ms
Bole average: 301.894ms
Debug average: 361.052ms
LogLevel average: 330.394ms
Pino average: 246.336ms
PinoAsync average: 129.507ms
PinoNodeStream average: 276.479ms
```

Reply:
Good spot. I had the wrong node version on, these were with Node 10. I'll update the numbers.